iris/tests/download_manager/open_download_folder.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from iris.api.helpers.download_manager_utils import download_file, DownloadFiles, downloads_cleanup
+from iris.test_case import *
+
+
+class Test(BaseTest):
+
+    def __init__(self):
+        BaseTest.__init__(self)
+        self.meta = 'Open download folder.'
+        self.test_case_id = '99480'
+        self.test_suite_id = '1827'
+        self.locales = ['en-US']
+        self.exclude = Platform.LINUX
+        self.blocked_by = '1513494'
+
+    def run(self):
+        navigate('https://www.thinkbroadband.com/download')
+
+        # Download a small file.
```

Comment:
Please remove this comment. The file to download has a very suggestive name so the comment will just make the code heavy to read.